sdk/storage/storage-blob/src/generated/src/models/mappers.ts

```diff
@@ -1311,14 +1285,14 @@ export const PageList: coreHttp.CompositeMapper = {
       },
       clearRange: {
         xmlName: "ClearRange",
-        xmlElementName: "ClearRange",
+        xmlElementName: "PageRange",
```

Comment:
Manually changing this to ClearRange fixes the test, unfortunately I'm not able to change this through autorest transform :(

Reply:
So I guess the response from the service has this result within the <ClearRange> tag and possibly we cannot ask to deserialize it into PageRange type.

We might have to use new types in convenience layer then. What do you think?  Defining the response type and handle the converting.  There's also a suggestion to use `Range` (src/Range.ts) for the results. Will that involve too much work?